bindings/python/iota_sdk/types/payload.py

```diff
@@ -10,15 +10,40 @@
 
 
 class PayloadType(Enum):
+    """Type of block payload.
+
+    Attributes:
+        TreasuryTransaction (4): TreasuryTransaction payload
+        TaggedData (5): TaggedData payload
+        Transaction (6): Transaction payload
+        Milestone (7): Milestone payload
+    """
     TreasuryTransaction = 4
     TaggedData = 5
     Transaction = 6
     Milestone = 7
 
 
 class Payload():
-    def __init__(self, type,  milestone: Optional[Any] = None, tagged_data=None, transaction=None, treasury_transaction: Optional[Any] = None):
+    """Base class for block payloads.
+
+    Attributes:
+        type (int): Payload type
+        milestone (optional): Milestone payload
+        tagged_data (optional): TaggedData payload
+        transaction (optional): Transaction payload
+        treasury_transaction (optional): TreasuryTransaction payload
+        
+    """
+    def __init__(self, type, milestone=None, tagged_data=None, transaction=None, treasury_transaction=None):
         """Initialize a payload
+
+        Args:
+            type (int): Payload type
+            milestone (optional): Milestone payload
+            tagged_data (optional): TaggedData payload
+            transaction (optional): Transaction payload
+            treasury_transaction (optional): TreasuryTransaction payload
```

Comment:
types

Reply:
yeah ... that went into `part 2` (see previous commit message :stuck_out_tongue_closed_eyes: )